helios-core/src/main/kotlin/helios/instances/instancesJava.kt

```diff
@@ -127,3 +128,36 @@ interface MapDecoderInstance<A, B> : Decoder<Map<A, B>> {
   }
 
 }
+
+interface EnumEncoderInstance<E : Enum<E>> : Encoder<Enum<E>> {
+
+  override fun Enum<E>.encode(): Json = JsString(name)
+
+  companion object {
+    operator fun <E : Enum<E>> invoke(): Encoder<Enum<E>> =
+      object : EnumEncoderInstance<E> {}
+  }
+}
+
+interface EnumDecoderInstance<E : Enum<E>> : Decoder<Enum<E>> {
```

Comment:
Those instances are for the type `kotlin.Enum` so they should be on `instances.kt`